internal/cli/command/config/config.go

```diff
@@ -25,6 +25,7 @@ func NewCmd(fileUtil file.File) *cobra.Command {
 		Short:   "Configure octavius client",
 		Long:    "This command helps configure client with control plane host, email id and access token",
 		Example: "octavius config [flags]",
+		Args:    cobra.MaximumNArgs(0),
```

Comment:
Just see if ExactArgs will be a better choice in steaf of MaximumNArgs. You can use whatever is more semantic.